WcaOnRails/app/jobs/dump_developer_database.rb

```diff
@@ -3,12 +3,17 @@
 class DumpDeveloperDatabase < SingletonApplicationJob
   queue_as :default
 
-  def perform
+  def perform(force_export: false)
     # Create developer database dump every 3 days.
     last_developer_db_dump = Timestamp.find_or_create_by(name: 'developer_db_dump')
-    if last_developer_db_dump.not_after?(3.days.ago)
-      Rake::Task["db:dump:development"].invoke
-      last_developer_db_dump.touch :date
+    if force_export || last_developer_db_dump.not_after?(3.days.ago)
+      running_on_dev_dump = Timestamp.exists?(name: DatabaseDumper::DEV_TIMESTAMP_NAME)
+
+      unless running_on_dev_dump
+        Rake::Task["db:dump:development"].invoke
+        last_developer_db_dump.touch :date
+        Rake::Task["db:dump:public_results"].reenable
```

Comment:
reenable is called for public_results in this file & `dump_public_results_database.rb`. But invoke was to the file it belongs to in both. I don't know the logic behind it, if it's intentional, please mark this as resolved.

Reply:
Whoops, indeed a copy/paste mistake :see_no_evil: Thanks for catching!